src/header/common/range.rs

```diff
@@ -0,0 +1,199 @@
+use std::fmt::{self, Display};
+use std::str::FromStr;
+
+use header::{Header, HeaderFormat, RangeUnit};
+use header::parsing;
+
+/// The `Range` header defined by HTTP/1.0.
```

Comment:
Yeah, the HTTP/1.0 was a mistake on my side. 

I've used `Pragma` header as a template and it uses `///` for the comments. 
Can you tell me why `#doc` is preferred? IMO, it's a lot noisier and harder to read.